java/flight/flight-core/src/test/java/org/apache/arrow/flight/TestFlightService.java

```diff
@@ -150,7 +150,9 @@ public FlightInfo getFlightInfo(CallContext context, FlightDescriptor descriptor
       Exception e =
           assertThrows(
               FlightRuntimeException.class, () -> client.getSchema(FlightDescriptor.path("test")));
-      assertEquals("No schema is present in FlightInfo", e.getMessage());
+      assertEquals(
```

Comment:
Can we do some sort of `assertTrue(e.getMessage().contains(...))`?

Reply:
Right. I will update.